### PR TITLE
add Microsoft datajs lib

### PR DESCRIPTION
**Pull request by hiddenboox**
Add Microsoft datajs-1.1.3 lib

### CODE REVIEW

**Comment by guybedford** [comment]
Thanks, can you make sure to include this in the correct alphabetical position?

**Comment by hiddenboox** [comment]
@guybedford now should be ok

File: registry.json [hunk 1]
```diff
@@ -177,6 +177,7 @@
   "d3": "github:mbostock/d3",
   "c3": "npm:c3",
   "datatables": "github:DataTables/DataTables",
+  "datajs": "github:hiddenboox/datajs",
```
Comment (guybedford) [inline registry.json]: To be pedantic, this should come before datatables.

**Comment by guybedford** [comment]
Not quite yet :)

File: registry.json (hunk 1 above)
Comment (aluanhaddad) [inline registry.json]: @hiddenboox Yeah, just move the entry and this will be good to go

**Comment by hiddenboox** [comment]
Ok, at now i can't, tommorow i will do that

**Pull request closed without merging**